worlds/generic/docs/setup_en.md

```diff
@@ -47,39 +50,142 @@ slot name and, if the relevant game requires it, files to associate it with that
+
+#### From a locally generated game
+
+After generating a game, a compressed folder will be output to the `/output` directory. Go to the
+[Archipelago Host Game Page](/uploads), click on `Upload File`, navigate to your Archipelago installation, and select
+the generated folder. This will create a new seed page using the information from this folder.
+
+### Hosting on a local machine
+
+The `.archipelago` file may be extracted from the compressed file. Double-clicking the file will then open
+`ArchipelagoServer.exe` in order to host the multiworld on the local machine. Alternatively, running
+`ArchipelagoServer.exe` and pointing the resulting file selection prompt to the `.archipelago` file or the generated
+compressed folder that was generated will begin hosting.
+
+## Connecting to an Archipelago Server
+
+The actual method of connection will vary depending on the game, so follow that game's setup guide, but all games will
+use the same general connection info noted here.
+
+### Connection Info
+
+For connecting from the game to the server, the connection info is needed for any of the game clients. Games that use
+data files will usually contain the connection info within these files, when hosted on the Archipelago website. If the
+information needs to be entered manually, it is usually comprised of four different sections.
 
-The `.archipelago` file may be run locally in order to host the multiworld on the local machine. This is done by
-running `ArchipelagoServer.exe` and pointing the resulting file selection prompt to the `.archipelago` file that was
-generated.
\ No newline at end of file
+* `Host Name` is the domain or IP address of the server. If the game is being hosted on the main Archipelago website
+this will be `archipelago.gg`. If the game is being hosted on your own local machine `localhost` will work. If the game
+is being hosted on another person's computer then you enter that person's public IP address.
+* `Port` is which port on the domain or IP address the game is being hosted on. On the website room pages, this is
+displayed as `archipelago.gg:<port>`. Most clients will accept that information being entered directly as is. If the
+information needs to be entered separately, then the port is the sequence of numbers after the `:`, and the `:` does
+not need to be entered. If a game is being hosted from the `ArchipelagoServer.exe`, this will default to `38281` but may
+be changed in the `host.yaml`.
+* `Slot Name` is the name of your player slot that you are connecting to. This is the same as the name that was set
+when creating your [YAML file](#creating-a-yaml). If the game is hosted on the website, this is also displayed on the
+room page. The name is caps sensitive.
+* `Password` is the password set by the host in order to join the multiworld. By default, this will be empty and is almost
+never required, but is possible to be set when generating the game. Generally, leave this field blank when it exists,
```

Comment:
"is possible to be set" is a somewhat awkward construction and could be changed to "one can be set"